evalai/utils/common.py

```diff
@@ -31,6 +33,34 @@ def convert(self, value, param, ctx):
             )
 
 
+def upload_presigned_url_file(file_name, presigned_url):
+    """
+    Function to upload a file, given the target presigned s3 url
+
+    Arguments:
+        file_name (str) -- the path of the file to be uploaded
+        presigned_url (str) -- the presigned url to upload the file on s3
+    """
+    echo(
+        style(
+            "Uploading the submission file...",
```

Comment:
@KhalidRmb we are showing `Uploading the submission file...` message for both test annotations and submissions. Please fix this

Reply:
Fixed this @Ram81